src/context.rs

```diff
@@ -622,7 +622,8 @@ impl Context {
 
     pub fn select_font_face(&self, family: &str, slant: FontSlant, weight: FontWeight) {
         unsafe {
-            ffi::cairo_select_font_face(self.0, CString::new(family).unwrap().as_ptr(), slant, weight)
+            let family = CString::new(family).unwrap();
+            ffi::cairo_select_font_face(self.0, family.as_ptr(), slant, weight)
```

Comment:
Nice catch, these are actually causing memory unsafety in practice

Reply:
All thanks for clippy